script.js

```diff
@@ -0,0 +1,201 @@
+// All the DOM selectors stored as short variables
+const chat = document.getElementById("chat");
+const inputWrapper = document.getElementById("input-wrapper");
+const input = document.getElementById("input");
+const sendBtn = document.getElementById("send");
+
+let questionNumber = 1;
+
+const botReply = (msg) => {
+  showMessage(msg, "bot");
+};
+
+const userReply = (msg) => {
+  showMessage(msg, "user");
+};
+
+const showMessage = (message, sender) => {
+  if (sender === "user") {
+    chat.innerHTML += `
+      <section class="user-msg">
+        <div class="bubble user-bubble">
+          <p>${message}</p>
+        </div>
+        <img src="assets/user.png" alt="User" />  
+      </section>
+    `;
+  } else if (sender === "bot") {
+    chat.innerHTML += `
+      <section class="bot-msg">
+        <img src="assets/bot.png" alt="Bot" />
+        <div class="bubble bot-bubble">
+          <p>${message}</p>
+        </div>
+      </section>
+    `;
+  }
+  chat.scrollTop = chat.scrollHeight;
+};
+
+const nextQuestion = (message) => {
```

Comment:
Super cool that you constructed this next question section using conditional statements. This enhances readability so much.